gui/layers/_visual_wrapper.py

```diff
@@ -84,6 +87,22 @@ def opacity(self, opacity):
 
         self._node.opacity = opacity
 
+    @property
+    def blending(self):
+        """string: Selects blending mode. Can be one of ('opaque',
+        'translucent', 'additive').
+        """
+        return self._blending
+
+    @blending.setter
+    def blending(self, blending):
+        if blending not in self._blending_modes:
+            raise ValueError("""blending must be between one of
```

Comment:
This will add extra indentation on your next line; try:
```python
raise ValueError('expected one of '
                 "{'opaque', 'translucent', 'additive'}; "
                 f'got {blending}')
```

Reply:
ah ok, will fix too